custom/icds/location_reassignment/parser.py

```diff
@@ -301,6 +303,13 @@ def _get_new_parent_site_codes(self, location_type_code):
                     parent_site_codes.add(new_location_details['parent_site_code'])
         return parent_site_codes
 
+    def valid_transitions_json(self):
```

Comment:
I did not actually, I found `attr` used in the code base and thought it was a good fit.
If there are advantages of using `JsonObject` and is the better standard to be followed, i am happy to change,
I know i had to add the logic to convert to JSON so that celery can keep it as arguments successfully because it could not serialise it. I searched too if i could just add a method on Transition that would help celery serialize it but could not find anything. I think JsonObject might provide but is there any other advantage?

Reply:
The serialization for celery was the main thing I was thinking of. `JsonObject` is used much more broadly in HQ, but I don't have strong opinions on it. I just asked in another comment, but I'd like to hear more about what `attrs` is doing for you. If it's more than `asdict` is probably worthwhile. If not, it might be better to use `JsonObject` just because it's more common.

Reply:
responded on the other comment.